### ISSUE TITLE
Setting the text for TextView programatically doesn't show the Title Label

### ISSUE THREAD

**Issue by vishiswoz** [CosmicMind/MaterialKit#117]
I have a TextView where I set the text programmatically like this:
```objective-c
[textView setText:@"Test"];
```
The problem is that I'm unable to get the Title Label to show up. I can get the Title Label to show up if I type text using the keyboard, but it won't show up if I set the text of the TextView programmatically, as shown  above.

I tried adding this line.
```objective-c
[[textView titleLabel] setHidden:NO];
```
But the Title Label still doesn't show up

Also I wanted to point out when I set the text of the TextView programmatically, only the `textWillProcessEdit` method of the `TextDelegate` runs, not the `textDidProcessEdit` method.

**Comment by danieldahan** [comment]
Yes, I see the issue. I am fixing it and will post an update shortly. Thank you!

**Comment by danieldahan** [comment]
The fix to this issue should be resolved in 1.27.12. If the issue persists, please reopen or create a new issue. Thank you!
